contrib/babelfishpg_tsql/src/backend_parser/gram-tsql-epilogue.y.c

```diff
@@ -789,19 +789,19 @@ is_json_query(List *name)
 }
 
 /*
-* Parse T-SQL CONTAINS predicate. Currently supports 
-* ... CONTAINS(column_name | (column_list), '<contains_search_condition>') ...
+* Parse T-SQL FREETEXT predicate. Currently supports 
+* ... CONTAINS({column_name | (column_list)}, '<contains_search_condition>') ...
+* ... FREETEXT({column_name | (column_list)}, '<freetext_search_condition>') ...
 * This function transform it into a Postgres AST that stands for
-* to_tsvector(pgconfig, column_name) @@ to_tsquery(pgconfig, babelfish_fts_rewrite('<contains_search_condition>'))
+* to_tsvector(pgconfig, column_name) @@ to_tsquery(pgconfig, <translated_search_condition>)
 * for column_list
-* to_tsvector(pgconfig, col_1) @@ to_tsquery(pgconfig, babelfish_fts_rewrite('<contains_search_condition>') OR
-* to_tsvector(pgconfig, col_2) @@ to_tsquery(pgconfig, babelfish_fts_rewrite('<contains_search_condition>') OR
-* ...
-* to_tsvector(pgconfig, col_n) @@ to_tsquery(pgconfig, babelfish_fts_rewrite('<contains_search_condition>')
-* where pgconfig = babelfish_fts_contains_pgconfig('<contains_search_condition>')
+* to_tsvector(pgconfig, col_1) @@ to_tsquery(pgconfig, <translated_search_condition>) OR
+* to_tsvector(pgconfig, col_2) @@ to_tsquery(pgconfig, <translated_search_condition>) OR
+* where pgconfig = babelfish_fts_contains_pgconfig('<search_condition>')
+* <translated_search_condition> is the modified <search_condition> in Postgres format
```

Comment:
Let's also add examples of these translated queries for both clauses in the comment section as well.